frontend_lib/i18next.scanner/fr/translation.json

```diff
@@ -65,5 +65,7 @@
   "YB": "Yo",
   "Recipient's email addresses": "Adresse email du ou des destinataires",
   "To add multiple recipients, separate the email addresses with a comma, a semicolon or a line break.": "Pour ajouter plusieurs destinataires, séparez les adresses email par une virgule, un point virgule ou un retour à la ligne.",
-  "Password": "Mot de passe"
+  "Password": "Mot de passe",
+  "Uploaded files": "Fichiers téléversés",
+  "You have not yet chosen any files to upload.": "Vous n'avez pas encore sélectionné de fichier à téléverser."
```

Comment:
I think "de fichier" should be plural but I'm not sure, can you confirm please ?

Reply:
Yes I think you are right because it isn't a specific file but any files

Reply:
I need the proof

Reply:
'Files' are countable so it should be plural in french ([see](https://www.francaisfacile.com/exercices/exercice-francais-2/exercice-francais-61719.php))